packages/contracts-bedrock/scripts/upgrades/holocene/templates/bundle_template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "chainId": $L1_CHAIN_ID,
+  "metadata": {
+    "name": "Holocene Hardfork - Contracts Upgrade",
+    "description": "Upgrades the `SystemConfig.sol`, `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "Upgrade `SystemConfig` proxy",
+        "description": "Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters."
+      },
+      "to": "$PROXY_ADMIN_ADDR",
+      "value": "0x0",
+      "data": "$TX_1_PAYLOAD",
+      "contractMethod": {
```

Comment:
If you want to simplify things, I'd be ok with deleting the metadata here that doesn't get used, i.e. the `contractMethod` and `contractInputsValues` fields. The minimum required fields for each object in the `transactions` array is `to` and `data`. The more human-readable `contractMethod` and `contractInputsValues` fields are never used since what really matters at the end of the day is the state diff, which is based on `to` and `data`

Reply:
They're used by `msup`, which is included in the image, to render the overview of the `superchain-ops` task.